### PR TITLE
JANITORIAL: ICB: Fix typos in comments

### PR DESCRIPTION
**Pull request by raziel-**
Question:

Regarding "cant", "dont", "didnt", "doesnt" typos

There are a lot of those still lingering in warning, error and message texts
Do you prefer to leave them as is, or should i go ahead and fix them as well?

Thank you

### CODE REVIEW

**Comment by raziel-** [comment]
i have no idea what went wrong...I didn't even touch the ultima engine(?)

**Comment by bluegr** [comment]
@raziel- messages shown in the console (warnings, errors etc) can be fixed without issues

**Comment by bluegr** [comment]
> ScummVM-iOS

I don't see any changes to the Ultima engine

File: engines/icb/animation_mega_set.cpp [hunk 1]
```diff
@@ -281,7 +281,7 @@ bool8 _vox_image::Init_custom_animation(const char *anim) {
 
 	// check for no weapon being set
 	if (custom == FALSE8 /*__NONE*/) {
-// custom must be in the current weapon set - bah, shouldn't have done it like this - its daft
+// custom must be in the current weapon set - bah, shouldn't have done it like this - it's daft
 		// rav (or equiverlant always come from pcp directory...
```
Comment (raziel-) [inline engines/icb/animation_mega_set.cpp:285]: should that be equivalent there?
Also missing a closing parenthesis in that comment

Comment (bluegr) [reply engines/icb/animation_mega_set.cpp]: Indeed. Done

**Comment by bluegr** [comment]
Thanks! Squashing

**Pull request merged**